### ISSUE TITLE
failed provisioning [Failed to execute operation: No such file or directory]

### ISSUE THREAD

**Issue by PI-Victor** [kubic-project/terraform#14]
whenever i try to provision a new cluster, i run into this issue, which is quite cryptic:
```
libvirt_domain.master (remote-exec): >>> Setting Salt master to dashboard.k8s.local
libvirt_domain.master (remote-exec): >>> Copying the Salt config
libvirt_domain.master (remote-exec): '/tmp/salt/config/minion.d/mine.conf' -> '/etc/salt/minion.d/mine.conf'
libvirt_domain.master (remote-exec): '/tmp/salt/config/minion.d/minion.conf' -> '/etc/salt/minion.d/minion.conf'
libvirt_domain.master (remote-exec): '/tmp/salt/grains' -> '/etc/salt/grains'
libvirt_domain.master (remote-exec): >>> Enabling & (re)starting the Salt minion
libvirt_domain.master (remote-exec): Failed to execute operation: No such file or directory
libvirt_domain.master (remote-exec): >>> FATAL: could not enable the kubelet
Error applying plan:

1 error(s) occurred:

* Script exited with non-zero exit status: 1

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```
not quite sure what file/dir it's talking about
cc:
@inercia @ereslibre

**Comment by inercia** [comment]
@PI-Victor I guess you didn't have the `kubelet` installed. Could you try again with the latest version from master? We included some checks in #13 for skipping this if you don't have the `kubelet` package installed.

**Comment by PI-Victor** [comment]
@inercia yes, kubelet was my first thought, but i checked and it was running, fortunately, it was my system and not a bug. it's running ok, now. thanks
